internal/factsengine/adapters/rabbitmq.go

```diff
@@ -0,0 +1,79 @@
+package adapters
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/wagslane/go-rabbitmq"
+)
+
+const (
+	gatherFactsExchanage string = "gather_facts"
+	factsExchanage       string = "facts"
+)
+
+type rabbitMQAdapter struct {
+	consumer  rabbitmq.Consumer
+	publisher *rabbitmq.Publisher
+}
+
+func NewRabbitMQAdapter(factsEngineService string) *rabbitMQAdapter {
```

Comment:
Could we expose the rabbitMQAdapter struct?, generally it's good pratice to expose the struct if we have some function that returns that struct, this does not means that the field should be exposed but only the struct itself. This issue is very common, and there are some linting rules that can be used in order to avoid that. Like the default rules of golangci-lint

Reply:
Done!